boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -82,14 +83,17 @@
 			   <11 0 &gpio1 7 0>,	/* D5 */
 			   <12 0 &gpio1 8 0>,	/* D6 */
 			   <13 0 &gpio1 9 0>,	/* D7 */
+
 			   <14 0 &gpio1 10 0>,	/* D8 */
 			   <15 0 &gpio1 11 0>,	/* D9 */
 			   <16 0 &gpio1 12 0>,	/* D10 */
 			   <17 0 &gpio1 13 0>,	/* D11 */
 			   <18 0 &gpio1 14 0>,	/* D12 */
 			   <19 0 &gpio1 15 0>,	/* D13 */
-			   <20 0 &gpio1 2 0>,	/* D14 */
-			   <21 0 &gpio1 3 0>;	/* D15 */
+			   /* GND */
+			   <20 0 &gpio0 27 0>,	/* AREF|D14 */
+			   <21 0 &gpio1 2 0>,	/* D15 */
+			   <22 0 &gpio1 3 0>;	/* D16 */
```

Comment:
This is not correct. See https://github.com/zephyrproject-rtos/zephyr/blob/f2c49619d2698a1abbeab62e5596a676938c863f/dts/bindings/gpio/arduino-header-r3.yaml#L18-L31